### PR TITLE
lxc: update to 5.0.1

### PR DESCRIPTION
**Pull request by CameronNemo**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/lxc/template [hunk 1]
```diff
@@ -26,13 +26,10 @@ make_dirs="
 pre_configure() {
 	# make sure the build uses the new docbook2x format,
 	# even though the command has the old "docbook2man" name
-	sed -e 's,xdocbook2man,xforcedocbook2x,g' -i configure.ac
-	./autogen.sh
+	vsed -e "s,sgml2man.full_path(),'defnot-docbook2man',g" -i meson.build
```
Comment (sgn) [inline srcpkgs/lxc/template:29]: Can you write a patch instead?

Comment (CameronNemo) [reply srcpkgs/lxc/template:29]: Yeah should not be a problem. Still need to figure out what the heck is happening on musl though.

Comment (sgn) [reply srcpkgs/lxc/template:29]: Please also patch https://github.com/lxc/lxc/blob/master/meson.build#L119-L128 to
```
distrosysconfdir = join_paths(sysconfdir, 'default')
conf.set('LXC_DISTRO_SYSCONF', distrosysconfdir)
```

Comment (sgn) [reply srcpkgs/lxc/template:29]:
```
distrosysconfdir = join_paths(sysconfdir, 'default')
conf.set('LXC_DISTRO_SYSCONF', distrosysconfdir)
```
Is the fix for musl

Comment (CameronNemo) [reply srcpkgs/lxc/template]: Added a patch for docbook2x.

Regarding musl/distrosysconfdir: can't I also `mkdir -p /etc/default` in pre_configure? That worked alright for me. We would not be able to upstream much I else I think.

Comment (sgn) [reply srcpkgs/lxc/template]: I would prefer to not touch anything outside of builddir.

Comment (CameronNemo) [reply srcpkgs/lxc/template]: thanks... that is how I was going to solve it but have not had the time ;)

**Pull request closed without merging**